source/leuven/services/openondemand.rst

```diff
@@ -273,14 +273,11 @@ The same applies for other choices of partitions on Genius or wICE clusters.
 JupyterLab
 -----------
 
-.. note::
-
-   This section is prepared for the scheduled migration of JupyterLab on
-   Thursday 25 July 2024. Some features will be available only after the migration.
-
-With this app you can create or (re)run Jupyter Notebooks. This can be handy both for R and Python coding.
-One of the benefits of using JupyterLab is the flexibility it offers in customizing user environments,
-which we explain below.
+With this app you can write and run
+`Jupyter <official JupyterLab documentation>`_ notebooks containing
+annotated Python or R code (among other languages). IPython consoles are
```

Comment:
It would be useful to explicitly add 'Julia' to this list, too. At the moment, searching for Julia returns nothing, but in this way, a curious Julia user receives a hint that Jupyter also supports that.